Jellyfin.Api/Auth/CustomAuthenticationHandler.cs

```diff
@@ -43,18 +44,17 @@ protected override Task<AuthenticateResult> HandleAuthenticateAsync()
             try
             {
                 var authorizationInfo = _authService.Authenticate(Request);
-                if (authorizationInfo == null)
+                var role = UserRoles.User;
+                // UserId of Guid.Empty means token is an apikey.
+                if (authorizationInfo.UserId.Equals(Guid.Empty) || authorizationInfo.User.HasPermission(PermissionKind.IsAdministrator))
```

Comment:
Doesn't this mean that if you remove the user id from the auth token that you become admin?

Reply:
The UserId is always pulled from the database, so that isn't an issue